src/main/java/htsjdk/variant/bcf2/BCF2GenotypeFieldDecoders.java

```diff
@@ -196,7 +196,7 @@ private final void generalDecode(final List<Allele> siteAlleles,
                         gt.add(getAlleleFromEncoded(siteAlleles, encode));
 
                     gb.alleles(gt);
-                    final boolean phased = (encoded[0] & 0x01) == 1;
+                    final boolean phased = ((encoded.length>1?encoded[1]:encoded[0]) & 0x01) == 1;
```

Comment:
Add some spaces between the clauses in the ternary expression, it would make it easier to read.